frontend/config/environment.js

```diff
@@ -54,7 +54,7 @@ module.exports = function(environment) {
   if (environment === "test") {
     // Testem prefers this...
     ENV.locationType = "none";
-
+    ENV.APP.autoboot = false;
```

Comment:
Is this really needed? I am assuming it fixed your `same root element` error?
what happens if you stop splitting the person-form test into 2 different modules?